app/templates/partials/questions/relationship.html

```diff
@@ -1,26 +1,25 @@
 {% import 'macros/helpers.html' as helpers %}
+<div class="question">
+  <fieldset class="question__fieldset">
+    <legend class="question__legend">
```

Comment:
The title should be in a `h2` and also hidden `legend` field. Take a look at the other question templates for the structure.